slang-antlr/src/test/java/com/sonarsource/slang/antlr/SLangParserTest.java

```diff
@@ -42,4 +51,20 @@ public void testFile() throws IOException {
     assertThat(context.children.isEmpty(), is(false));
   }
 
+  @Test
+  public void testBinaryExpression() {
+    SLangLexer lexer = new SLangLexer(CharStreams.fromString("x = 1\n//comment\ny = 2 + \"1\""));
+    CommonTokenStream tokens = new CommonTokenStream(lexer);
+    SLangParser parser = new SLangParser(tokens);
+    SLangParser.SlangFileContext tree = parser.slangFile();
+    ParseTreeWalker walker = new ParseTreeWalker();
+
+    for (Token token : tokens.getTokens()) {
+      System.out.println(token.getLine() + ": " + token.getText());
+    }
+
+    assertThat(tree.children, notNullValue());
+    assertThat(tree.children.isEmpty(), is(false));
```

Comment:
we switched to assertj for assertion to be more verbose

Reply:
more "explicit".